doc/actor.md

```diff
@@ -7,570 +8,521 @@ http://stixproject.github.io/data-model/1.2/ta/ThreatActorType/
 
 * This entry is required
 
-### Keyword Key
+* Keyword Key
+  * Plumatic Schema: :id
 
-* Plumatic Schema: :id
-
-### String Value
-
-The URI of this entity.
-
-* Plumatic Schema: Str
+* String Value
+  * The URI of this entity.
+  * Plumatic Schema: Str
 
 ## MapEntry: :type -> String
```

Comment:
Yes I believe that if you could "protect" the keyword it would look better. But the current version looks good to me.